pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -128,6 +131,29 @@ func (ct *cdcTester) startStatsCollection() func() {
 	}
 }
 
+type AuthorizationRuleKeys struct {
+	PrimaryConnectionString string `json:"primaryConnectionString"`
+}
+
+func getEventHubConnectionString() (string, error) {
+	// These have all been configured via the Azure Portal.
+	cmdStr := "az eventhubs namespace authorization-rule keys list --resource-group cdc-testing --namespace-name roachtest-cdc --name roachtest"
```

Comment:
Do you know what kind of config settings we have on this event hub wrt to data retention?
Can we guarantee that this test will not run cross availability zones (since this might be $)?

Reply:
Right now I have it set to deleting after 1h.  Good point on cross-az, its in US East right now, I asked #test-eng about the region of nightlies and I'll change the location accordingly once they respond